MentorMultiThreading/MentorMultiThreading/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Win32.SafeHandles;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MentorMultiThreading
+{
+    internal class Program
+    {
+        static int[] numberArray;
+        static int firstSum=0;
+        static ManualResetEvent resetEvent=new ManualResetEvent(false);
+        
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array");
+            int size =Convert.ToInt16(Console.ReadLine());
+            Console.WriteLine("Enter the array of integers (space-separated):");
+            string input = Console.ReadLine();
+            numberArray = Array.ConvertAll(input.Split(' '), int.Parse);
+            int midPoint = numberArray.Length / 2;
+            //First Thread
+            ThreadPool.QueueUserWorkItem(new WaitCallback(SumOperation), new LimitValues(0, midPoint));
+            //SecondThread
+            ThreadPool.QueueUserWorkItem(new WaitCallback(SumOperation), new LimitValues(midPoint, size));
```

Comment:
```suggestion
            ThreadPool.QueueUserWorkItem(SumOperation, new LimitValues(0, midPoint));
            //SecondThread
            ThreadPool.QueueUserWorkItem(SumOperation, new LimitValues(midPoint, size));
```
could have simplified like above? See if this works. Same for below line as well.